### PR TITLE
Custom feed feature

### PR DESCRIPTION
**Pull request by jefim**
#55 
#56

### CODE REVIEW

File: .github/workflows/build_main.yml [hunk 1]
```diff
@@ -1,168 +1,200 @@
...
+    - name: Test
+      run: dotnet test --collect:"XPlat Code Coverage"
+      working-directory: "${{ env.workdir }}"
+
+    - name: Check Code Coverage
+      id: check_code_coverage
+      shell: pwsh
+      run: |
+        Get-ChildItem -Path "." -Include "coverage.cobertura.xml" -Recurse | Copy-Item -Destination .
+        [double] $coverage = ([xml]($tmp = Get-Content -Path 'coverage.cobertura.xml')).SelectSingleNode("/coverage/packages/package[@name='${{ env.workdir }}']").Attributes['line-rate'].Value
```
Comment (RikuVirtanen) [inline .github/workflows/build_main.yml:162]: Does this disable the use of forward slash in workdir?

Comment (jefim) [reply .github/workflows/build_main.yml:162]: I think that according to all our guidelines we should not have any slashes (ie sobfolders) in working dir though?

Comment (RikuVirtanen) [reply .github/workflows/build_main.yml:162]: The reason for the need for forward slashes is referring to the issue with Code Quality Check using the workdir to determine what to check inside the repository. When we have multiple similarly starting Tasks the Code Quality Check tries to check every folder with that filter. For example Frends.HTTP.Request and Frends.HTTP.RequestBytes. Of course I can try to fix this in the workflow itself.

File: .github/workflows/build_test.yml [hunk 2]
```diff
@@ -66,20 +70,46 @@ jobs:
           OUTPUT_DETAILS: detailed
           VALIDATE_CSHARP: true
           IGNORE_GITIGNORED_FILES: true
-          FILTER_REGEX_INCLUDE: "${{ inputs.workdir }}"
+          FILTER_REGEX_INCLUDE: "${{ inputs.workdir }}/.*"
```
Comment (RikuVirtanen) [inline .github/workflows/build_test.yml:73]: ${{ inputs.workdir }}/.* might not work for this. I think you need to remove the dot from the filter in order for the filter to work correctly.

File: .github/workflows/linux_build_test.yml [hunk 3]
```diff
@@ -66,20 +70,46 @@ jobs:
         OUTPUT_DETAILS: detailed
         VALIDATE_CSHARP: true
         IGNORE_GITIGNORED_FILES: true
-        FILTER_REGEX_INCLUDE: "${{ inputs.workdir }}"
+        FILTER_REGEX_INCLUDE: "${{ inputs.workdir }}/.*"
```
Comment (RikuVirtanen) [inline .github/workflows/linux_build_test.yml:73]: Same here. ${{ inputs.workdir }}/.* might not work for this. I think you need to remove the dot from the filter in order for the filter to work correctly.

**Review by RikuVirtanen — APPROVED** [review]

**Pull request merged**